### PR TITLE
add list footer

### PR DESCRIPTION
**Pull request by omrihecht**
add list footer component
move clear action to footer.

multiList:
image

multiSelect:
image

---

singleList:
image

singleSelect:
image

### CODE REVIEW

File: projects/ui-framework/src/lib/form-elements/lists/list.scss [hunk 1]
```diff
@@ -63,3 +63,14 @@
     }
   }
 }
+
+@mixin selectPanel {
+  display: block;
+  box-sizing: border-box;
+  border: 1px solid $grey-500;
+  border-top: none;
+  border-bottom-right-radius: 4px;
```
Comment (galloween) [inline projects/ui-framework/src/lib/form-elements/lists/list.scss]: we have a variable for border-radius now

Comment (omrihecht) [reply projects/ui-framework/src/lib/form-elements/lists/list.scss]: where?

Comment (omrihecht) [reply projects/ui-framework/src/lib/form-elements/lists/list.scss]: i saw, maybe we should move it to shapes?

Comment (galloween) [reply projects/ui-framework/src/lib/form-elements/lists/list.scss]: not sure. if someone will be looking for variable for something, the first (and probably only) place he will check will be the variables file....

**Pull request merged**